flow-builder/Dockerfile

```diff
@@ -12,8 +12,9 @@ RUN set -ex ; \
     openssh-client~=8 \
     py-openssl~=19 \
     maven~=3 \
-    su-exec~=0.2 \
     shadow~=4 \
+    su-exec~=0.2 \
+    uchardet~=0.0.6-r0 \
```

Comment:
The idea of using `~=` is to not specify the full package version, just the *interesting* part of it. e.g. `uchardet~=0.0.6` 
If alpine maintainers make a fix in the package, even if is the same upstream version will increase the `rX` suffix, e.g. `uchardet~=0.0.6-r1`

That's called *fuzzy version matching*:

https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management#Holding_a_specific_package_back

Reply:
Yes, I was too quick. Good catch!